### PR TITLE
Vm   refactor new dev

### PR DESCRIPTION
**Pull request by PizzaCat07**

### CODE REVIEW

File: src/components/SearchResults.js [hunk 1]
```diff
@@ -121,12 +105,12 @@ const SearchResults = ({searchQueryTerm, filterArray}) => {
...
-    console.log('fetch Db');
+    //console.log('fetch Db');
     return getDbData(searchResults);
   }, [searchResults]);
 
   const transformData = (crimCodeDbData, mvaDbData) => {
-    console.log('transform data');
+    //console.log('transform data');
     //create subsection for crime code renders
     var subsectionData = createSubSectionArray(crimCodeDbData);
```
Comment (VLKH3) [inline src/components/SearchResults.js:115]: When this is refactored again, please change these vars to lets

File: src/components/SearchResults.js [hunk 2]
```diff
@@ -121,12 +105,12 @@ const SearchResults = ({searchQueryTerm, filterArray}) => {
   );
 
   const fetchDB = useMemo(() => {
```
Comment (doolaik) [inline src/components/SearchResults.js:107]: Is this being used anywhere? Linting says no

Comment (PrismStudent07) [reply src/components/SearchResults.js:107]: useMemo runs function on first render, then will only re-run if choosen variables are changed. Need to update lint

File: src/components/SearchResults.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (PizzaCat07) [reply src/components/SearchResults.js]: changed

File: src/components/SearchResults.js (hunk 2 above)
Comment (PizzaCat07) [reply src/components/SearchResults.js:107]: useMemo runs function on first render, then will only re-run if choosen variables are changed. Need to update lint

File: src/components/SearchResults.js [hunk 3]
```diff
@@ -144,18 +128,57 @@ const SearchResults = ({searchQueryTerm, filterArray}) => {
       mvaRenderData: mvaRenderData,
     });
   };
-  //console.log(renderObject);
 
   const setRenderData = useMemo(() => {
```
[the remark this answers is not in the document]
Comment (PizzaCat07) [reply src/components/SearchResults.js:132]: useMemo runs function on first render, then will only re-run if choosen variables are changed. Need to update lint

**Pull request merged**